pkg/util/install/utils.go

```diff
@@ -37,7 +37,9 @@ func ApplyDefaultLabels(meta *v1.ObjectMeta, component string, name string) {
 func ApplyServiceDefaults(service *corev1.Service, component string, name string) {
 
 	ApplyDefaultLabels(&service.ObjectMeta, component, name)
-	service.Spec.Selector = CreateDefaultLabels(nil, component, name)
+	if service.Spec.Selector == nil {
```

Comment:
I think that is a good idea. From what I saw in other examples, the way to detect if a resource is newly created was:

~~~go
if existing.CreationTimestamp.IsZero() {
  // do new stuff
}
~~~

Reply:
@ctron Thanks, updated PR with that.